interface/forms/procedure_order/new.php

```diff
@@ -403,13 +423,13 @@ function validate(f) {
 ?>
 </p>
 
-<center>
+
 
 <p>
-<table border='1' width='95%' id='proctable'>
+<table  width='100%' id='proctable' class="table">
```

Comment:
I've been playing around with bootstrap on another PR and the class table is needed for bootstrap ui styling:
https://github.com/openemr/openemr/pull/587